src/Hedgehog/Property.fs

```diff
@@ -149,6 +150,9 @@ module private Pretty =
             (renderAndDiscards discards)
             |> append sb
 
+        sprintf "--- Seed to recheck this case: %d" seed.Value
```

Comment:
It'd be nice to have this *after* the shrinked value (at the very end) so that we're consistent with what we do in the Haskell version.

Also, instead of

```
--- Seed to recheck this case: 11194825392996116235
```

it'd be nice to give to the user the full command to run, e.g.

```
This failure can be reproduced by running:
> Property.recheck (Size 0) (Seed 6047214807766778671 8250247924578451779) prop_failure
```

Reply:
I added nearly the string requested, `Seed` is a record, and F# doesn't support the syntax used in the above message. When you used `prop_failure` was that meant to be taken literally? Not sure how else to identify the property if that wasn't your intention.

Reply:
No, it's just an example from what we do in Haskell. I should have mentioned it. `prop_failure` is the name of the failing property.

Reply:
We can replace it with `<property>` for example, or just omit it for now.

Reply:
Do properties have names in the F# version?